kernel/kernel-api/src/main/java/io/delta/kernel/internal/TransactionImpl.java

```diff
@@ -336,8 +339,15 @@ private long commitWithRetry(
             commitAsVersion = rebaseState.getLatestVersion() + 1;
             dataActions = rebaseState.getUpdatedDataActions();
             domainMetadatas = Optional.of(rebaseState.getUpdatedDomainMetadatas());
```

Comment:
one qn: why not load the latest CRC as part of the conflict detection and rebase state construction?

Reply:
Good suggestion updated